questions/Poker_game.md

```diff
@@ -0,0 +1,7 @@
+# Pick a card
+
+## Question
+
+Welcome to **divine**! Here is the problem statement:
+
+You are playing a poker game and the final round that could cost you millions of dollars is to pick a card from a list of cards which is arranged in ascending order that contains a number that is given to you in the shortest amount of time, which way would you take to pick that card and in the shortest amount of time possible so that you can cash out millions of dollars!!!!
```

Comment:
I would rewrite this question. It isn't very clear on what it is asking. Why do you have to pick a card quickly?

```suggestion
You are playing a card game. This final round could win you millions of dollars. You are given a card number and a list of cards. The list is in ascending order. Your task is to find the card number you are given in that list of cards in the shortest time possible. If you're too slow you will not win.
```

Reply:
I think the change of framing @OfErosAngel suggested is a great idea. Poker is not usually done in this way so moving to just "a card game" makes this more intuitive.